lib/internal/backend/file/file.go

```diff
@@ -53,15 +53,15 @@ func NewFileBackend(file *os.File, debug, info log.Logger) *FileBackend {
 	}
 }
 
-func (lb *FileBackend) GetImageInfo(dockerURL string) ([]string, *common.ParsedDockerURL, error) {
+func (lb *FileBackend) GetImageInfo(dockerURL string) ([]string, string, *common.ParsedDockerURL, error) {
```

Comment:
this return signature is getting messy, it is not obvious what `[]string, string` is. Can we document or name the return values?